internal/service/encryptionatrest/resource_test.go

```diff
@@ -133,139 +116,57 @@ func TestAccEncryptionAtRest_basicAzure(t *testing.T) {
-			AzureEnvironment:         conversion.StringPtr("AZURE"),
-			SubscriptionID:           conversion.StringPtr(os.Getenv("AZURE_SUBSCRIPTION_ID")),
-			ResourceGroupName:        conversion.StringPtr(os.Getenv("AZURE_RESOURCE_GROUP_NAME_UPDATED")),
-			KeyVaultName:             conversion.StringPtr(os.Getenv("AZURE_KEY_VAULT_NAME_UPDATED")),
-			KeyIdentifier:            conversion.StringPtr(os.Getenv("AZURE_KEY_IDENTIFIER_UPDATED")),
-			Secret:                   conversion.StringPtr(os.Getenv("AZURE_SECRET_UPDATED")),
-			TenantID:                 conversion.StringPtr(os.Getenv("AZURE_TENANT_ID")),
-			RequirePrivateNetworking: conversion.Pointer(false),
-		}
-
-		azureKeyVaultUpdatedAttrMap = convertToAzureKeyVaultAttrMap(&azureKeyVaultUpdated)
-	)
-
-	resource.Test(t, resource.TestCase{
-		PreCheck:                 func() { acc.PreCheck(t); acc.PreCheckEncryptionAtRestEnvAzureWithUpdate(t) },
-		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
-		CheckDestroy:             testAccCheckMongoDBAtlasEncryptionAtRestDestroy,
-		Steps: []resource.TestStep{
-			{
-				Config: acc.ConfigEARAzureKeyVault(projectID, &azureKeyVault, false),
-				Check: resource.ComposeAggregateTestCheckFunc(
-					testAccCheckMongoDBAtlasEncryptionAtRestExists(resourceName),
-					resource.TestCheckResourceAttr(resourceName, "project_id", projectID),
-					testCheckResourceAttr(resourceName, "azure_key_vault_config.0", azureKeyVaultAttrMap),
-					resource.TestCheckResourceAttr(datasourceName, "project_id", projectID),
-					testCheckResourceAttr(datasourceName, "azure_key_vault_config", azureKeyVaultAttrMap),
-				),
-			},
-			{
-				Config: acc.ConfigEARAzureKeyVault(projectID, &azureKeyVaultUpdated, false),
-				Check: resource.ComposeAggregateTestCheckFunc(
-					testAccCheckMongoDBAtlasEncryptionAtRestExists(resourceName),
-					resource.TestCheckResourceAttr(resourceName, "project_id", projectID),
-					testCheckResourceAttr(resourceName, "azure_key_vault_config.0", azureKeyVaultUpdatedAttrMap),
-					resource.TestCheckResourceAttr(datasourceName, "project_id", projectID),
-					testCheckResourceAttr(datasourceName, "azure_key_vault_config", azureKeyVaultUpdatedAttrMap),
-				),
-			},
-			{
-				ResourceName:      resourceName,
-				ImportStateIdFunc: testAccCheckMongoDBAtlasEncryptionAtRestImportStateIDFunc(resourceName),
-				ImportState:       true,
-				ImportStateVerify: true,
-				// "azure_key_vault_config.0.secret" is a sensitive value not returned by the API
-				ImportStateVerifyIgnore: []string{"azure_key_vault_config.0.secret"},
-			},
-		},
-	})
-}
-
-func testCheckResourceAttr(resourceName, prefix string, attrsMap map[string]string) resource.TestCheckFunc {
-	checks := acc.AddAttrChecksPrefix(resourceName, []resource.TestCheckFunc{}, attrsMap, prefix)
-
-	return resource.ComposeAggregateTestCheckFunc(checks...)
-}
-
-func TestAccEncryptionAtRest_azure_requirePrivateNetworking_preview(t *testing.T) {
```

Comment:
Had to move this test to `encryptionatrestprivateendpoint/resource_test.go` since it was updated to use MONGODB_ATLAS_PROJECT_EAR_PE_ID (atlas project with PE flag enabled). Since these tests use resources (project, key vault, etc) common with private endpoint tests, keeping them in one package so tests can run sequentially.